pkg/service/storage/handlers/blob/accept.go

```diff
@@ -67,36 +63,27 @@ func Accept(ctx context.Context, s AcceptService, req *AcceptRequest) (*AcceptRe
 		}
 	} else {
 		// locate the piece from the pdp service
-		pdpPiece, err := s.PDP().PieceFinder().FindPiece(ctx, req.Blob.Digest, req.Blob.Size)
+		pdpPiece, found, err := s.PDP().API().ResolvePiece(ctx, req.Blob.Digest)
```

Comment:
We won't be able to accept until commp is calculated if we do this. We should just verify we have it stored as we do above for non-PDP.